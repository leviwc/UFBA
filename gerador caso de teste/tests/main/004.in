26 179
20 22
26 9
5 20
1 19
12 24
5 16
5 21
10 18
17 16
18 19
12 3
3 10
18 23
7 20
23 16
24 22
23 20
13 19
11 21
21 12
11 3
21 20
16 7
24 20
3 14
11 13
18 20
14 19
22 10
4 22
4 16
24 6
3 5
14 24
9 7
7 21
10 7
4 25
23 10
5 15
14 16
17 21
5 1
10 1
25 5
12 18
22 16
3 15
9 2
19 17
7 15
20 1
9 6
26 17
21 18
4 26
4 24
16 26
11 7
13 2
20 4
1 7
15 18
20 25
18 5
24 1
6 11
7 23
20 11
11 1
6 25
1 16
16 8
18 4
2 21
25 7
9 16
26 25
6 17
25 3
12 7
9 21
9 25
5 23
22 2
14 14
1 22
2 19
11 14
1 2
23 1
3 1
15 12
24 5
25 16
25 2
14 9
17 13
2 4
7 19
15 22
8 24
5 14
4 8
23 6
25 13
10 21
13 4
1 9
26 11
4 19
8 5
10 15
3 22
23 22
26 1
12 14
19 23
18 17
21 8
3 17
13 6
21 22
14 25
20 3
15 19
15 9
2 2
2 3
4 4
1 21
5 7
2 8
1 4
22 14
9 19
17 9
23 3
11 9
2 10
26 10
16 12
8 11
7 2
1 1
18 7
11 18
12 12
15 16
5 12
15 15
26 2
9 22
19 6
5 2
8 18
7 4
12 2
13 22
22 25
26 24
5 26
12 19
3 8
9 9
24 9
2 24
11 17
25 11
12 10
10 20
6 22
22 17
22 12
7 13
3 19
26 22
23 26
11 24
