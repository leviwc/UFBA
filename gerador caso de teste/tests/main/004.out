Resultado com busca binaria: 6
629.917000 s
Resultado com backtracking apenas: 6
0.150000 s
Resultado testando todas as possibilidades: 6
2396.131000 s
Resultado com busca binaria otimizada: 6
266.338000 s
